katharsis-rs/src/main/java/io/katharsis/rs/KatharsisFilter.java

```diff
@@ -100,8 +109,10 @@ public void filter(ContainerRequestContext requestContext) throws IOException {
         try {
             dispatchRequest(requestContext);
         } catch (WebApplicationException e) {
+        	e.printStackTrace();
             throw e;
         } catch (Exception e) {
+        	e.printStackTrace();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1148)